cs.py

```diff
@@ -213,6 +218,14 @@ def main():
         key, value = option.split('=', 1)
         kwargs[key].add(value.strip(" \"'"))
 
+    if 'region' not in flags or not isinstance(flags['region'], str):
+        flags['region'] = 'cloudstack'
```

Comment:
With the flags / options split you can also do `options.setdefault('region', 'cloudstack')` or `region = options.get('region', 'cloudstack')` to avoid the `if` check.